airflow/hooks/druid_hook.py

```diff
@@ -31,14 +34,18 @@ class DruidHook(BaseHook):
     :type timeout: int
     :param max_ingestion_time: The maximum ingestion time before assuming the job failed
     :type max_ingestion_time: int
+    :param druid_broker_conn_id: The connection id to the Druid broker machine
+    :type druid_broker_conn_id: string
     """
     def __init__(
             self,
             druid_ingest_conn_id='druid_ingest_default',
             timeout=1,
-            max_ingestion_time=None):
+            max_ingestion_time=None,
+            druid_broker_conn_id='druid_broker_default'):
 
         self.druid_ingest_conn_id = druid_ingest_conn_id
+        self.druid_broker_conn_id = druid_broker_conn_id
```

Comment:
It's impossible to guess this but we try to created an entry for all defaults hooks in `airflow/utils/db.py` so that users can find and edit the default connection in the list.

Reply:
thanks. will add an entry :)

Reply:
thanks. Will add an entry into the db table .